fdbclient/include/fdbclient/ClientKnobs.h

```diff
@@ -293,6 +293,7 @@ class ClientKnobs : public KnobsImpl<ClientKnobs> {
 
 	// Encryption-at-rest
 	bool ENABLE_ENCRYPTION_CPU_TIME_LOGGING;
+	int64_t EKP_TENANT_ID_TO_DROP; // Which tenant the the EKP should throw key not found in simulation
```

Comment:
Do we see value in making it a `list` or `vector`? 
KNOB can be a `comma separated` string..

Reply:
Sure it might be useful to make it a list for future use cases